homeassistant/components/system_bridge/__init__.py

```diff
@@ -328,3 +336,29 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 async def async_reload_entry(hass: HomeAssistant, entry: ConfigEntry) -> None:
     """Reload the config entry when it changed."""
     await hass.config_entries.async_reload(entry.entry_id)
+
+
+async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Migrate old entry."""
+    _LOGGER.debug("Migrating from version %s", config_entry.version)
+
+    if config_entry.version == 1 and config_entry.minor_version == 1:
+        # Migrate to CONF_TOKEN, which was added in 1.2
+        new_data = dict(config_entry.data)
+        new_data.setdefault(CONF_TOKEN, config_entry.data.get(CONF_API_KEY))
+        new_data.pop(CONF_API_KEY, None)
```

Comment:
We should never remove an item when doing minor version migrations. Otherwise it's not a minor version migration.